packages/web3-shared/evm/src/libs/PayloadEditor.ts

```diff
@@ -88,6 +88,7 @@ export class PayloadEditor {
 
     private getRawConfig() {
         const { method, params } = this.payload
+        const coder = ABICoder as unknown as ABICoder.AbiCoder
```

Comment:
Make a lib here: `packages/web3-shared/evm/src/libs/AbiCoder.ts` for one-time type fixing.